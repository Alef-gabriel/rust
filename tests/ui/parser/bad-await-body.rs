fn main() {
    await{}()
    //~^   ERROR: cannot find struct, variant or union type `await` in this scope
}
