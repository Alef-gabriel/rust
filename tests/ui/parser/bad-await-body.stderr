error[E0422]: cannot find struct, variant or union type `await` in this scope
  --> $DIR/bad-await-body.rs:2:5
   |
LL |     await{}()
   |     ^^^^^ not found in this scope

error: aborting due to previous error

For more information about this error, try `rustc --explain E0422`.
